sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -388,7 +388,7 @@ object ScalaNativePluginInternal {
         librt ++ libunwind ++ linked.links
           .map(_.name) ++ garbageCollector(gc).links
       }
-      val linkopts  = links.map("-l" + _) ++ linkingOpts
+      val linkopts  = links.map("-l" + _) ++ linkingOpts ++ Seq("-lpthread")
```

Comment:
Shouldn't be necessary once we add `@link("pthread")` on the extern object for pthread.

Reply:
process_monitor.c doesn't compile on the travis server without this.